source/_includes/installation/core.md

```diff
@@ -102,3 +106,10 @@ If this address doesn't work you may also try `http://localhost:8123` or `http:/
 When you run the `hass` command for the first time, it will download, install and cache the necessary libraries/dependencies. This procedure may take anywhere between 5 to 10 minutes. During that time, you may get "site cannot be reached" error when accessing the web interface. This will only happen for the first time, and subsequent restarts will be much faster.
 
 </div>
+{% if page.installation_type == 'linux' or page.installation_type == 'generic-x86-64' %}
+
+#### Run Homeassistant core on startup
+
+On modern linux distros you can follow this [community guide](https://community.home-assistant.io/t/autostart-using-systemd/199497) to autostart homeassistant on boot.
```

Comment:
We have explicitly left this out of scope for our documentation, and consider this out of scope for our documentation. There are just too many variants available. This installation method is an advanced one, those users should have no problem with this at all.

Nevertheless, thanks for willing to contribute 👍

Reply:
Ok, you are right!